tests/test_jemx_plugin.py

```diff
@@ -0,0 +1,69 @@
+import pytest
+import logging
+import requests
+import json
+
+from tests.test_basic import ask
+
+logger = logging.getLogger(__name__)
+
+default_params = dict(instrument='jemx',
+                        jemx_num='2',
+                        product_type='jemx_image',
+                        scw_list=['010200230010.001'],
+                        osa_version='OSA10.2',
+                        detection_threshold=5.0,
+                        radius=15.,
+                        query_type='Real')
+
+dummy_params = dict(
+    query_status="new",
+    query_type="Dummy",
+    instrument="jemx",
+    async_dispatcher=False
+)
+
+
+def test_default(dispatcher_live_fixture):
+    server = dispatcher_live_fixture
+
+
+@pytest.mark.jemx_plugin
+@pytest.mark.depends(on=['test_default'])
```

Comment:
actual syntax is this one:

```python
@pytest.mark.dependency(depends=["test_default"])
```

https://pytest-dependency.readthedocs.io/en/stable/usage.html